extension.js

```diff
@@ -177,7 +179,12 @@ class ArchUpdateIndicator extends PanelMenu.Button {
 
 		if (!USE_BUILDIN_ICONS && St.Settings.get().gtk_icon_theme) {
 			let theme = new Gtk.IconTheme();
-			theme.set_custom_theme( St.Settings.get().gtk_icon_theme );
+
+			if (gtkVersion >= 4) {
+				theme.set_theme_name( St.Settings.get().gtk_icon_theme );
+			} else {
+				theme.set_custom_theme( St.Settings.get().gtk_icon_theme );
```

Comment:
This doesn't work for me, returning error "set_custom_theme is not a function". It works if I change it to "gtk_icon_theme_set_custom_theme". ArchLinux, gnome-shell:44rc-1 via FCGU (https://gitlab.com/fabiscafe/fcgu) repo.